src/main/java/net/imagej/launcher/ClassLauncher.java

```diff
@@ -80,10 +80,23 @@ public static void main(final String[] arguments) {
 		// property.
 		if (Boolean.getBoolean("imagej.splash")) {
 			try {
-				SplashScreen.show();
-			}
-			catch (Throwable t) {
-				t.printStackTrace();
+				URLClassLoader classLoader = ClassLoaderPlus.getInImageJDirectory(null, "jars/imagej-launcher.jar");
```

Comment:
I expect that Java 11 is going to hate this with "illegal reflective access". I think we can avoid it if we scan for splash icons using a strategy that does not rely on the classpath. Perhaps we can just inspect the available JAR files manually and rip out the resources using `java.util.zip`.